Web3-Builders-Alliance/Albedo
# Feature/landing page UI

### Summary

Integrated Material UI for the FAQ section and resolved Git conflicts. Improved landing page's FAQ section and general layout.

### Details

1. Integrated Material UI to implement an FAQ Accordion for the Landing Page.
2. Resolved Git conflicts and merged changes from the master branch into the feature branch.

### How to Test

1. Navigate to the Landing Page.
2. Test the FAQ Accordion by clicking on the questions to reveal the answers.
3. Check for console errors or warnings.
